src/renderers/WebGLRenderer.js

```diff
@@ -1147,6 +1151,61 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
+	this.renderOverride = function ( overrideMaterial, scene, camera, renderTarget, clearColor, clearAlpha ) {
+
+		var originalClearColor = this.getClearColor(), originalClearAlpha = this.getClearAlpha(), originalAutoClear = this.autoClear;
+
+		this.autoClear = false;
```

Comment:
I like everything in this PR except this clear logic here. I would let the user handle the clear on their own...